src/Module.Server/Rewards/CrpgRewardClient.cs

```diff
@@ -55,8 +56,12 @@ private void HandleRewardUser(CrpgRewardUser message)
             (Color color, string verb) = gain > 0
                 ? (new Color(65, 105, 225), "Gained")
                 : (new Color(0.74f, 0.28f, 0.01f), "Lost");
+            string upkeepMessage = message.LowPopulation
+                ? "free , low population server"

```

Comment:
There is no space before commas but one after -> "free, low..."

Reply:
done